### PR TITLE
added basic and contextual panels

### PR DESCRIPTION
**Pull request by brianbodollo**
leave a comment

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,30 @@
+GEM
```
Comment (srph) [inline Gemfile.lock:1]: Remove this file for now

**Comment by brianbodollo** [comment]
screenshot from 2015-09-12 20 06 32
screenshot from 2015-09-12 20 06 26

**Comment by srph** [comment]
- You can edit the top comment instead of making a new one. I'll take a look at the code

File: styles/components/panel.scss [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+			display: block;
+			padding: 10px 15px;
+			margin-bottom: -1px;
+			background: $brand-white;
+			border: 1px solid $border-color;
+		}		
+	}
+}
+
+.panel-default {
```
Comment (srph) [inline styles/components/panel.scss]: - We're not following bootstrap's conventions. We're following [**rscss**](https://github.com/rstacruz/rscss).

File: styles/components/panel.scss [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+	border: 2px solid $brand-gray;
+	.panel-header {
+		padding: 10px 15px;
+		color: $brand-black;
+		background: #f5f5f5;
+		border-bottom: 1px solid $brand-gray;
+		border-top-left-radius: 3px;
```
Comment (srph) [inline styles/components/panel.scss]: You can use `$boder-radius-base` instead.

File: styles/components/panel.scss [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+		padding: 10px 15px;
+		color: $brand-black;
+		background: #f5f5f5;
+		border-top: 1px solid $brand-gray;
+		border-bottom-left-radius: 3px;
+		border-bottom-right-radius: 3px;
+	}
+}
+
+.panel-sky {
```
Comment (srph) [inline styles/components/panel.scss]: Why do you keep on repeating the styles every time? We have modifiers for this reason.

File: styles/components/panel.scss [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+		padding: 10px 15px;
+		color: $brand-white;
+		background: $brand-crimson;
+		border-top: 1px solid $brand-crimson;
+		border-bottom-left-radius: 3px;
+		border-bottom-right-radius: 3px;
+	}
+}
+
+.table {
```
Comment (srph) [inline styles/components/panel.scss]: You shouldn't be having this in here. Plus, you're overriding the `.table` class when it's supposedly in the `table` file.

File: styles/components/panel.scss [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+		border-bottom-right-radius: 3px;
+	}
+	th {
+		font-weight: bold;
+		vertical-align: bottom;
+		border-bottom: 2px solid $border-color;
+		padding: 8px;
+		line-height: $line-height-base;
+		text-align: left;
+	}
```
Comment (srph) [inline styles/components/panel.scss:130]: Space between two selectors? How about that?

``` sass
// bad
td {
}
th {
}

// good
td {
}

th {
}
```

File: styles/components/panel.scss [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+		padding: 10px 15px;
+		color: $brand-white;
+		background: $brand-sun;
+		border-top: 1px solid $brand-sun	;
+		border-bottom-left-radius: 3px;
+		border-bottom-right-radius: 3px;
+	}
+}
+
+.panel-crimson {
```
Comment (srph) [inline styles/components/panel.scss]: This, too.

File: styles/components/panel.scss [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+		padding: 10px 15px;
+		color: $brand-white;
+		background: $brand-emerald;
+		border-top: 1px solid $brand-emerald;
+		border-bottom-left-radius: 3px;
+		border-bottom-right-radius: 3px;
+	}
+}
+
+.panel-sun {
```
Comment (srph) [inline styles/components/panel.scss]: Dude?

File: styles/components/panel.scss [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+		padding: 10px 15px;
+		color: $brand-white;
+		background: $brand-sky;
+		border-top: 1px solid $brand-sky;
+		border-bottom-left-radius: 3px;
+		border-bottom-right-radius: 3px;
+	}
+}
+
+.panel-emerald {
```
Comment (srph) [inline styles/components/panel.scss]: No.

File: styles/components/panel.scss (hunk 8 above)
Comment (srph) [inline styles/components/panel.scss]: This, too.

File: styles/components/panel.scss (hunk 9 above)
Comment (srph) [inline styles/components/panel.scss]: This.

File: styles/components/panel.scss [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+		vertical-align: bottom;
+		border-bottom: 2px solid $border-color;
+		padding: 8px;
+		line-height: $line-height-base;
+		text-align: left;
+	}
+	td {
+		padding: 8px;
+		vertical-align: top;
+		border-top: 1px solid $border-color;
```
Comment (srph) [inline styles/components/panel.scss]: Since when did we have `$border-color`?

File: styles/components/panel.scss (hunk 9 above)
Comment (srph) [inline styles/components/panel.scss]: This.

**Comment by srph** [comment]
You should check out other files as well, @maebe22. Here's an example

``` sass
.panel {
  > .panel-heading {
    // 
  }

  > .panel-body {
    // 
  }

// whatever

  &.-sky {
    color: blue; // whatever
  }

  &.-sky > .panel-body {
    color: blue; // whatever
  }

 // other variations
}
```

**Pull request closed without merging**